src/intl/en/page-developers-index.json

```diff
@@ -44,8 +44,8 @@
   "page-developers-js-libraries-link": "JavaScript libraries",
   "page-developers-language-desc": "Using Ethereum with familiar languages",
   "page-developers-languages": "Programming languages",
-  "page-developers-learn": "Learn Ethereum development",
-  "page-developers-learn-desc": "Read up on core concepts and the Ethereum stack with our docs",
+  "page-developers-learn": "Ethereum development",
+  "page-developers-learn-desc": "Read up on core concepts and the Ethereum stack with our docs.",
```

Comment:
I would suggest reverting these changes since it will not fix the root problem. The font size can vary between browsers and devices. There is no guarantee that you will always have one line.

Reply:
Yeah. Makes sense.